resources/js/about/about.data.js

```diff
@@ -0,0 +1,78 @@
+import externalLinks from "../external_links/index.js";
+
+const formatLink = (link, label) => {
+  return `<a href=${link}>${label}</a>`;
+};
+
+const formattedLinks = {
```

Comment:
I have to ask: Why not put both href and html content in this file (why put the hrefs in another file?).
Also: If you do that, the entire structure of links will be small enough to just put in about.js, which would get rid of two files, right?

Reply:
Having a single file containing a list of links would make it easier to write a test that none are dead?